src/main/java/liquibase/ext/clickhouse/params/ParamsLoader.java

```diff
@@ -0,0 +1,73 @@
+package liquibase.ext.clickhouse.params;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.*;
+
+public class ParamsLoader {
+  private static Map<String, String> liquibaseClickhouseProperties = null;
+
+  private static Set<String> validProperties =
+      new HashSet<>(Arrays.asList("clusterName", "tableZooKeeperPathPrefix", "tableReplicaName"));
+
+  private static StringBuilder appendWithComma(StringBuilder sb, String text) {
+    if (sb.length() > 0) sb.append(", ");
+    sb.append(text);
+
+    return sb;
+  }
+
+  private static void checkProperties(Map<String, String> properties)
+      throws InvalidPropertiesFormatException {
+    StringBuilder errMsg = new StringBuilder();
+
+    for (String key : properties.keySet())
+      if (!validProperties.contains(key)) {
+        appendWithComma(errMsg, "unknown property: ").append(key);
+      }
+
+    if (errMsg.length() > 0 || properties.size() != validProperties.size()) {
+      appendWithComma(errMsg, "this properties should be defined: ");
+      StringBuilder vp = new StringBuilder();
+      for (String prop : validProperties) appendWithComma(vp, prop);
+      errMsg.append(vp);
+    }
+
+    if (errMsg.length() > 0) throw new InvalidPropertiesFormatException(errMsg.toString());
+  }
+
+  public static Map<String, String> getLiquibaseClickhouseProperties() {
+    if (liquibaseClickhouseProperties != null) return liquibaseClickhouseProperties;
+
+    String defaultsFile = "liquibaseClickhouse.properties";
+
+    Map<String, String> result = null;
+    File fileProps = new File(defaultsFile);
+    if (fileProps.exists()) {
+      try (FileInputStream stream = new FileInputStream(fileProps)) {
+        Properties props = new Properties();
+        props.load(stream);
+
+        result = new HashMap<String, String>();
+        for (Map.Entry<Object, Object> entry : props.entrySet())
+          result.put(entry.getKey().toString(), entry.getValue().toString());
+      } catch (FileNotFoundException e) {
+
+      } catch (IOException e) {
+
+      }
+    }
+
+    if (result != null) {
+      try {
+        checkProperties(result);
+      } catch (InvalidPropertiesFormatException e) {
+        e.printStackTrace();
```

Comment:
same here, maybe it is a good idea to recover, but we should use a proper logging backend (slf4j or something)